site/content/contribute/getting-started/code-review.md

```diff
@@ -38,47 +38,65 @@ If you are a core committer seeking a review
-    * Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
-    * If you are Mattermost staff, try to take into account the timeoff calendar.
-    * Try to avoid assigning the same person to all of your reviews unless they are related.
-    * When in doubt, ask for recommendations on our community server.
-4. Assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers) and label your pull request with `2: QA Review`.
-    * It is the PM or QA tester's responsibility to determine the scope of required testing, if any.
-    * Reviews by QA may occur at the same time as review by core committers. Be sure to ask for a second review as needed when changes are made.
-    * The QA tester should remove the `2: QA Review` label and add the `QA Review Done` label when their review is done, or if a review is deemed not necessary.
-5. Apply additional labels as necessary:
+2. Immediately assign `1: PM Review`, `2: Dev Review`, and `3: QA Review`.
+    * Your pull request should not be merged until these labels are later removed.
+3. Apply additional labels as necessary:
     * `CherryPick/Approved`: Apply this if the pull request is meant for a quality or patch release.
     * `Do Not Merge/Awaiting PR`: Apply this if the pull request depends on another (e.g. server changes)
     * `Setup Test Server`: Apply this to create a test server with your changes for review.
-    * See [labels](/contribute/getting-started/labels) for more details.
-6. Assign a milestone as necessary.
+    * See [labels](/contribute/getting-started/labels) for additional labels.
+4. Assign a milestone as necessary.
     * Most issues are targeted for an upcoming release, and should be assigned the corresponding milestone.
     * The milestone is mandatory for bug fixes that must be cherry-picked.
-7. Wait for a review.
+5. Request a review from a [product manager](/contribute/getting-started/core-committers/#product-managers).
+    * The choice of product manager is up to you.
+        - In most cases, choose the product mananger embedded with your team.
+        - If your change primarily touches another team's codebase, consider their product mananger.
+        - If in doubt, assign any product mananger and comment about the uncertainty. The product mananger may reassign as appropriate.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `1: PM Review` label only when their review is done.
```

Comment:
How do they communicate that? QA has the 'QA Review Done' label.

Reply:
Good question -- I had been assuming a "GitHub review" gets completed in this case, but it would be useful to call that out explicitly.